<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Финансы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#2f3237;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Продукты питания . . . . . . . . </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Продукты бытовые . . . . . . .</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Одежда . . . . . . . . . . . . . . . . .</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Здоровье / красота . . . . . . .</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Кредиты / долги . . . . . . . . .</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Развлечения и подарки . . . </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Транспорт (такси, бензин)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Связь (телефон, интертнет)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_14">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_15">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>230</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_16">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_17">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_18">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_19">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_20">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_21">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Резерв . . . . . . . . . . . . . . . . . </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: &quot;white&quot;</string>
    </property>
    <property name="text">
     <string>Зарплата . . . . . . . . . . . . . . . </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_22">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>440</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_23">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBookBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:'white'</string>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_21">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>469</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>106</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;orange&quot;;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
    border-bottom-left-radius: 10px; 
    border-bottom-right-radius: 10px</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QCalendarWidget QAbstractItemView:enabled 
  {
  	font-size:12px;  
  	color: rgb(180, 180, 180);  
  	background-color: #2f3237;  
  	selection-background-color: orange; 
  	selection-color: black; 
  }
 QCalendarWidget QMenu {
  	width: 90px;
  	left: 18px;
  	color: white;
  	font-size: 14px;
  	background-color: rgb(100, 100, 100);
}
QCalendarWidget QWidget { alternate-background-color:#26282b; }</string>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="createBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:'white'</string>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:'white'</string>
    </property>
    <property name="text">
     <string>Считать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
